src/tester/test-runner.ts

```diff
@@ -282,22 +282,22 @@ async function runCommand(log: LoggerWithErrors, cwd: string | undefined, cmd: s
 
 /** Returns all immediate subdirectories of the root directory that have changed. */
 export function gitChanges(diffs: GitDiff[]): PackageId[] {
-    const changedPackages = new Map<string, Set<DependencyVersion>>();
+    const changedPackages = new Map<string, Map<string, DependencyVersion>>();
```

Comment:
Now that `DependencyVersion` is no longer a single primitive, but rather an object, I had to replace the usage of `Set`.